src/Reflection/Mutation/AddClassMethod.php

```diff
@@ -0,0 +1,29 @@
+<?php
+declare(strict_types=1);
+
+namespace Roave\BetterReflection\Reflection\Mutation;
+
+use PhpParser\Node\Stmt\ClassMethod;
+use Roave\BetterReflection\Reflection\Mutator\ReflectionClassMutator;
+use Roave\BetterReflection\Reflection\ReflectionClass;
+
+class AddClassMethod
+{
+    /**
+     * @var ReflectionClassMutator
+     */
+    private $mutator;
+
+    public function __construct()
+    {
+        $this->mutator = new ReflectionClassMutator();
```

Comment:
Don't really like this lack of constructor injection: it has bitten us before, so I'd prefer to have an entry point like in https://github.com/Roave/BetterReflection/pull/337

Reply:
I marked the mutators as internal but ok :)